lib/wickedbase.pm

```diff
@@ -100,8 +100,8 @@ sub assert_wicked_state {
 sub reset_wicked {
     my $self = @_;
     # Remove any config file and leave the system clean to start tests
-    assert_script_run('find /etc/sysconfig/network/ -name "ifcfg-*" -not -name "ifcfg-lo" -exec rm {} \;');
-    assert_script_run('find /etc/sysconfig/network/ -name "routes" -o -name "ifroute-*" -exec rm {} \;');
+    assert_script_run('find /etc/sysconfig/network/ \( -name "ifcfg-*" -not -name "ifcfg-lo" \) -exec rm {} \;');
```

Comment:
can elaborate what is going on here ? why you encapsulate just two *arguments* into brackets ? when I try to run this command in bash I am getting : 
```
find /etc/sysconfig/network/ ( -name "ifcfg-*" -not -name "ifcfg-lo" ) -exec echo {};
bash: syntax error near unexpected token `('
```

from what I know `(` is creating subshell . what the point to create a subshell for `-name "ifcfg-*" -not -name "ifcfg-lo" ` ?

Reply:
you need to escape the `(` so it goes into `find` and is not interpreted by the shell...
from `man find`
```
   OPERATORS
       Listed in order of decreasing precedence:

       ( expr )
              Force precedence.  Since parentheses are special to  the  shell,
              you  will  normally need to quote them.  Many of the examples in
              this manual page use backslashes  for  this  purpose:  `\(...\)'
              instead of `(...)'.
```